docs/user-guide/replicasets/hpa-rs.yaml

```diff
@@ -3,11 +3,11 @@ kind: HorizontalPodAutoscaler
 metadata:
   name: frontend-scaler
 spec:
-  scaleRef:
+  scaleTargetRef:
     kind: ReplicaSet
     name: frontend
     subresource: scale
   minReplicas: 3
   maxReplicas: 10
   cpuUtilization:
-    targetPercentage: 50
\ No newline at end of file
+    targetPercentage: 50
```

Comment:
This will not work. Based on the data type definition on the code ``targetCPUUtilizationPercentage: 50`` is the only right way.